clusterloader2/pkg/measurement/util/runtimeobjects/runtimeobjects_test.go

```diff
@@ -426,13 +427,111 @@ func changeEnv(in *v1.Pod) *v1.Pod {
 	return out
 }
 
+func noWhitespace(s string) string {
+	s = strings.ReplaceAll(s, " ", "")
+	s = strings.ReplaceAll(s, "\n", "")
+	s = strings.ReplaceAll(s, "\r", "")
+	s = strings.ReplaceAll(s, " ", "") //non-breaking space
+	return strings.ReplaceAll(s, "\t", "")
+}
+
+const fullCompareErrorDifferentEnv = `Not matching templates, diff:   v1.PodSpec{
+    Volumes:        nil,
```

Comment:
Is there a way to make it shorter, e.g. by skipping lines that do match?

Reply:
Not really - we are limited by public function API, and it does not allow us to hack around defaultReporter. We could write our own Reporter, but it seems that it's not well documented and reserved for cmp.Equal. From [cmp.Reporter](https://pkg.go.dev/github.com/google/go-cmp/cmp#Reporter)

```
Reporter is an Option that can be passed to Equal. When Equal traverses the value trees, it calls PushStep as it descends into each node in the tree and PopStep as it ascend out of the node. The leaves of the tree are either compared (determined to be equal or not equal) or ignored and reported as such by calling the Report method.
```